src/index.ts

```diff
@@ -25,6 +25,7 @@ const main = async () => {
         "axelar-dojo",
         "bostrom",
         "core",
+        "doravota-testnet",
```

Comment:
@tomas-dora Could you remove this chainId?
It is test net that Keplr infra team running node.